modules/test/playwright/pages/batch-planner/dataMigrationCenter.page.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/applicationsMenu.page';
+
+export class DataMigrationCenterPage {
+	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly page: Page;
+	readonly newButton: Locator;
+	readonly entityTypeSelector: Locator;
+	readonly importFileMenuItem: Locator;
+	readonly importStrategySelector: Locator;
+	readonly fileSelector: Locator;
+	readonly nextButton: Locator;
+	readonly scopeSelector: Locator;
+	readonly startImportButton: Locator;
+	readonly updateStrategySelector: Locator;
+
+	constructor(page: Page) {
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+		this.page = page;
+		this.newButton = page.getByRole('button', {name: 'New'});
+		this.entityTypeSelector = page.getByLabel('Entity Type');
+		this.importFileMenuItem = page.getByRole('menuitem', {
+			exact: true,
+			name: 'Import File',
+		});
+		this.importStrategySelector = page.getByLabel('Import Strategy');
+		this.fileSelector = page.getByLabel('File ( .csv, .json, .jsonl)');
```

Comment:
will this locator work if we won't have the feature flag for CSV enabled?

Reply:
No, nice catch!